Source/Core/VideoCommon/TextureConverterShaderGen.cpp

```diff
@@ -38,32 +38,81 @@ ShaderCode GenerateShader(APIType api_type, const UidData* uid_data)
   if (api_type == APIType::OpenGL)
   {
     out.Write("SAMPLER_BINDING(9) uniform sampler2DArray samp9;\n"
+              "uniform float3 filter_coefficients;\n"
+              "uniform float gamma_rcp;\n"
+              "uniform float2 clamp_tb;\n"
```

Comment:
clamp_tb is unused

Reply:
Oh whoops, looks like I missed implementing the clamp in the efb-to-tex shaders..